### ISSUE TITLE
NTR: Indoleacetic acid increased in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8824]
New term request Indoleacetic acid increased in urine (segal_080313143919):   UMLS: 0476343

**Comment by pnrobinson** [comment]
@MickeySegal
please provide more context. I cannot find an exact match, only this: https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_64631

**Comment by MickeySegal** [comment]
I have this in Hartnup disease from a book I won't have available until I return on Thursday.  But there is similar information in https://www.jstage.jst.go.jp/article/tjem1920/91/4/91_4_383/_pdf/-char/en "an excessive urinary excretion of
indican and indoleacetic acid".
Also https://www.ncbi.nlm.nih.gov/books/NBK559076/ "Excess tryptophan present in the gut is converted to indolic compounds by the action of colonic bacteria. These indolic compounds are conjugated in the liver and are excreted in urine as indican"

We also have "Indican increased in urine" (segal_080313143816).

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000332
- label: Elevated urinary indoleacetic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: PMID:4228120
- def: The amount of indoleacetic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8824
- synonyms:
